### PR TITLE
Performance

### PR DESCRIPTION
**Pull request by afmhenry**
also have done quick sample for performance--what do you think? Unsettled amounts PR first :)

### CODE REVIEW

**Comment by afmhenry** [comment]
@basgroot  ready for review, applied same updates as we talked about for UA.

File: account-history/performance/index.html [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+            id="idHrefRetrieveToken">from here</a> to the box below:</span><br />
+        <input type="text" class="token-field" id="idBearerToken" value="" placeholder="Paste token here.."
+          autofocus /></label>
+      <input type="button" class="action-button" value="Validate" id="idBtnValidate" /><br />
+      <label>
+        Account: <select class="selector" id="idCbxAccount" <option value="-">Click Validate to populate</option>
+        </select>
+      </label>
+    </div>
+    <h1>Unsettled Amounts</h1>
```
Comment (basgroot) [inline account-history/performance/index.html:34]: Shouldn't this be 'Performance', or 'Portfolio performance'?

Comment (basgroot) [reply account-history/performance/index.html:34]: Ah, you name it "Trade Performance".

Comment (afmhenry) [reply account-history/performance/index.html:34]: yup, missed it

**Review by basgroot — APPROVED** [review]

**Pull request merged**